workshop/exercise-5/README.md

```diff
@@ -1,47 +1,58 @@
+1. First, let's get the ingress subdomain for our cluster.
 
-    ```
-    knctl deploy \
-        --service fib-knative \
-        --git-url https://github.com/IBM/fib-knative \
-        --git-revision vnext \
-        --service-account build-bot \
-        --image registry.ng.bluemix.net/<NAMESPACE>/fib-knative:vnext \
-        --managed-route=false
-    ```
+	```
+	ibmcloud ks cluster-get <my-cluster-name>
+	```
 
-	This command will tell Knative to go out to github, find the code, build it into a container, and push that container to the IBM Container Registry. One thing you'll notice if you follow the output logs is that this deploy command also tags my app versions with a `latest` and a `previous` tag.
+	Example Output:
+	```
+	Ingress Subdomain:      bmv-knative.us-east.containers.appdomain.cloud   
+	```
+	Ingress is a Kubernetes service that balances network traffic workloads in your cluster by forwarding public or private requests to your apps. This Ingress Subdomain is an externally available and public URL providing access to your cluster. Take note of this Ingress Subdomain.
 
-2. See the revisions using knctl.
+2. Next, update the default URL for new Knative apps by editing the configuration:
 
 	```
-	knctl revisions list
+	kubectl edit cm config-domain --namespace knative-serving
 	```
-	Expected output:
-	
-	```
-    Revisions
 
-    Service      Name               Tags      Annotations  Conditions  Age  Traffic  
-    fib-knative  fib-knative-00002  latest    -            5 OK / 5    20h  100% -> fib-knative.default.bmv-knative.us-east.containers.appdomain.cloud  
-    ~            fib-knative-00001  previous  -            5 OK / 5    20h  -
+3. Change all instances of `example.com` to your ingress subdomain, which should look something like: `bmv-knative.us-east.containers.appdomain.cloud`. There should be two instances of `example.com`, one under `data` and one under `annotations`. New Knative applications will now be assigned a route with this host, rather than `example.com`.
+
+### Forward specific requests coming into IKS ingress to the Knative Ingress Gateway
 
-    2 revisions
+1. When requests come in to our fibonacci application through the ingress subdomain, we want them to be forwarded to the Knative ingress gateway. Update the `forward-ingress.yaml` file with your own ingress subdomain, prepended with `fib-knative.default`. Remember that the fully qualified domain name for a route has the following form: `{route}.{namespace}.{domain}`.
 
-    Succeeded
-    ```
+The file should look something like:
 
-3. Curl the application to see that 100% of the traffic is hitting your new fib-knative revision, starting the sequence with 0. Ensure you've updated the command with your own ingress subdomain.
+```yaml
+  apiVersion: extensions/v1beta1
+  kind: Ingress
+  metadata:
+    name: iks-knative-ingress
+    namespace: istio-system
+    spec:
+      rules:
+        - host: fib-knative.default.<ingress_subdomain>
```

Comment:
I though we were going to use a wildcard here so we would catch all ksvcs?

Reply:
this PR is starting to include too many issues. Let's open a new issue.

Reply:
https://github.com/IBM/knative101/issues/42